src/basic/string-util.h

```diff
@@ -53,6 +53,10 @@ static inline const char* true_false(bool b) {
         return b ? "true" : "false";
 }
 
+static inline const char* plus_minus(bool b) {
+        return b ? "+" : "-";
+}
```

Comment:
maybe the green check mark/red cross thing that "bootctl"'s feature display does is prettier? i mean, unicode ftw! colors ftw!

Reply:
Eh, I think +/- is more readable. People might want to also use `grep` on this. So I think it's better to keep as is.